dojo/db_migrations/0194_remove_usercontactinfo_slack_user_id.py

```diff
@@ -0,0 +1,16 @@
+# Generated by Django 4.1.11 on 2023-11-23 09:57
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
```

Comment:
maybe you could merge the two db migrations 0193 and 0194 together?

Reply:
Thanks. These migrations are in relevant separate commits.
Do you want me to merge all commits in this MR into one?

Reply:
Yeah, I guess it makes sense to merge the two db migrations.